### PR TITLE
Add "Get Android Studio" desktop launcher

### PR DESCRIPTION
**Pull request by cosimoc**
https://phabricator.endlessm.com/T26176

### CODE REVIEW

File: apps/com.google.AndroidStudio/eos-android-studio.desktop.in [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Name=Get Android Studio
+Exec=/usr/bin/eos-install-app-helper --app-id com.google.AndroidStudio --remote flathub --required-archs x86_64 %U
```
Comment (wjt) [inline apps/com.google.AndroidStudio/eos-android-studio.desktop.in]:
```suggestion
Exec=/usr/bin/eos-install-app-helper --app-id com.google.AndroidStudio --remote flathub --branch stable --required-archs x86_64 %U
```

See https://github.com/endlessm/eos-application-tools/pull/42 where I discovered this is needed.

**Review by wjt — CHANGES_REQUESTED** [review]

**Comment by wjt** [comment]
Please also add it to `po/POTFILES.in`.

File: apps/com.google.AndroidStudio/eos-android-studio.desktop.in [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+Terminal=false
+Icon=eos-android-studio
```
Comment (wjt) [inline apps/com.google.AndroidStudio/eos-android-studio.desktop.in:8]: Might be nice to add:

```suggestion
# TRANSLATORS: don't translate this
Icon=eos-android-studio
```

When I bought translations for “Get Unity”, several translators translated this string.

File: apps/com.google.AndroidStudio/eos-android-studio.desktop.in [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Name=Get Android Studio
```
Comment (wjt) [inline apps/com.google.AndroidStudio/eos-android-studio.desktop.in:4]:
```suggestion
# TRANSLATORS: "Android Studio" is the name of an application. Please do not translate it.
Name=Get Android Studio
```

**Review by wjt — CHANGES_REQUESTED** [review]
A couple of other things I thought of

**Comment by cosimoc** [comment]
@wjt Thanks... The file was already in `po/POTFILES.in`. I now also addressed your other suggestions.

**Review by wjt — APPROVED** [review]

**Comment by wjt** [comment]
Sorry, must have overlooked the POTFILES.in change.

**Pull request merged**